samples/bluetooth/mesh/light/src/model_handler.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/mesh/models.h>
+#include <dk_buttons_and_leds.h>
+#include "model_handler.h"
+
+#define BT_COM_ID_NORDIC 0x0059
```

Comment:
This should probably be moved to a public header file, although I cannot find an existing one that makes sense, create new one?